CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.9)
-project("cosim" VERSION "0.8.0")
+project("cosim" VERSION "0.10.0")
```

Comment:
`cosim-cli` version numbers do not follow `libcosim` version numbers. The last release was 0.7.0, so this next version should be 0.8.0.